tests/php/SiteTreeSubsitesTest.php

```diff
@@ -199,36 +224,36 @@ public function testCopyToSubsite()
 
         // Staff is shifted to top level and given a unique url segment
         $domain = $otherSubsite->domain();
-        $this->assertEquals('http://'.$domain.'/staff-2/', $staffPage2->AbsoluteLink());
-        $this->assertEquals('http://'.$domain.'/contact-us-2/', $contactPage2->AbsoluteLink());
+        $this->assertEquals('http://' . $domain . '/staff-2/', $staffPage2->AbsoluteLink());
+        $this->assertEquals('http://' . $domain . '/contact-us-2/', $contactPage2->AbsoluteLink());
     }
 
     public function testPageTypesBlacklistInCMSMain()
     {
-        $editor = $this->objFromFixture('Member', 'editor');
-        Session::set("loggedInAs", $editor->ID);
+        $editor = $this->objFromFixture(Member::class, 'editor');
+        Session::set('loggedInAs', $editor->ID);
 
         $cmsmain = new CMSMain();
 
-        $s1 = $this->objFromFixture('Subsite', 'domaintest1');
-        $s2 = $this->objFromFixture('Subsite', 'domaintest2');
+        $s1 = $this->objFromFixture(Subsite::class, 'domaintest1');
+        $s2 = $this->objFromFixture(Subsite::class, 'domaintest2');
 
-        $s1->PageTypeBlacklist = 'SiteTreeSubsitesTest_ClassA,ErrorPage';
+        $s1->PageTypeBlacklist = implode(',', [SiteTreeSubsitesTest_ClassA::class, ErrorPage::class]);
         $s1->write();
 
         Subsite::changeSubsite($s1);
         $hints = Convert::json2array($cmsmain->SiteTreeHints());
         $classes = $hints['Root']['disallowedChildren'];
-        $this->assertContains('ErrorPage', $classes);
-        $this->assertContains('SiteTreeSubsitesTest_ClassA', $classes);
-        $this->assertNotContains('SiteTreeSubsitesTest_ClassB', $classes);
+        static::assertContains(ErrorPage::class, $classes);
+        static::assertContains(SiteTreeSubsitesTest_ClassA::class, $classes);
+        static::assertNotContains(SiteTreeSubsitesTest_ClassB::class, $classes);
```

Comment:
The way it was was correct (for SilverStripe) - here's some context :D https://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-148407321 https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.static-vs-non-static-usage